### DISCUSSION TITLE
Question about this outfit

### DISCUSSION THREAD

**Discussion by Ant9cp** [Monika-After-Story/MonikaModDev#8583] [general - spritepacks]
Does anyone know how to get this outfit?
<img width="629" alt="Clipping" src="https://user-images.githubusercontent.com/71310908/148646481-96abd432-be5f-402e-b6bd-ce332d3a3252.PNG">

**Comment by SpaceHX** [comment]
Not sure where you can get that jacket. but I have found something similar.

https://www.mediafire.com/file/nezwojirwz7mq01/dnsgfs-jacket-leather.zip/file

You can download it there.

(Disclaimer: That jacket is not mine. It is by a person named dnsgfs on Reddit.

**Reply by Ant9cp** [reply]
The gloves aren't included

**Reply by SpaceHX** [reply]
> The gloves aren't included

Yeah, like I said, it's not the same, but it is similar.
